prisma/seed.ts

```diff
@@ -15,7 +15,7 @@ const seedRandomData = async () => {
     for (let i = 1; i <= 5; i++) {
         const difficulty = await prisma.difficulty.create({
             data: {
-                name: "ExampleParentDifficulty" + i,
+                name: "Difficulty" + i,
```

Comment:
The parent difficulty name needs to be a prefix of the child difficulty name for filtering. I've made that change.